src/routes/trainer.js

```diff
@@ -1,12 +1,14 @@
 const express = require('express');
 const { validateUpdate } = require('../validations/trainer');
 const trainerController = require('../controllers/trainer');
+const validation = require('../validations/trainer');
 
 const router = express.Router();
 
 router
   .put('/:id', validateUpdate, trainerController.updateTrainer)
```

Comment:
replace validateUpdate with valdiation.validateUpdate since it's now inside an object